File: Codility/Arrays/OddOccurrencesInArray.rb
def unpaired (arr)
	sorted = arr.sort
  p sorted
	sorted.each_with_index do |number, index|
		if index.even?
			if sorted[index] != sorted[index + 1]
        return number
      end
		end
	end
end

result = unpaired([1, 9, 8, 2, 9, 1, 8, 14, 2])
p result

File: Codility/Arrays/OddOccurrencesInArray.js
function unpaired (arr) {
  var sorted = arr.sort()
  var result = null
  sorted.forEach(function (number, index, array) {
    if (index % 2 === 0) {
      if (array[index] !== array[index + 1]) result = number
    }
  })
  return result
}

var result = unpaired([1, 9, 2, 9, 1, 7, 2])
console.log(result)
